x-pack/plugins/maps/public/embeddable/map_embeddable_factory.ts

```diff
@@ -91,14 +91,9 @@ export class MapEmbeddableFactory implements EmbeddableFactoryDefinition {
     input: MapEmbeddableInput,
     parent?: IContainer
   ) => {
-    const {
-      getInitialLayers,
-      getHttp,
-      MapEmbeddable,
-      mergeInputWithSavedMap,
-    } = await lazyLoadMapModules();
+    const { getHttp, MapEmbeddable, mergeInputWithSavedMap } = await lazyLoadMapModules();
     const savedMap = await this._fetchSavedMap(savedObjectId);
-    const layerList = getInitialLayers(savedMap.layerListJSON);
+    const layerList = JSON.parse(savedMap.layerListJSON);
```

Comment:
This change is not relevant to migrating client to NP. `getInitialLayers` should be used to create the layer list. Maybe we can talk about changing behavior but not in the scope of this PR.